DataStructures/LRUCache.playground/Sources/DoublyLinkedList.swift

```diff
@@ -182,6 +184,7 @@ public class DoublyLinkedList<T: Payload> {
 
     /// The head of the LinkedList starting at index 0
     /// Returns bool indicating wether or not the remove was successful
```

Comment:
This code comment isn't as valid anymore. Can it be updated to maybe `"discardableResult returns bool indicating whether or not the remove was successful"`? What're your thoughts?
This applies to lines `143` and `99` as well.

Reply:
I used the `- Returns:` syntax to get that styling in the Quick Help.